Sources/Image.swift

```diff
@@ -323,15 +323,20 @@ extension Kingfisher where Base: Image {
                     with: data,
                     scale: scale,
                     duration: 0.0,
-                    preloadAll: preloadAllGIFData,
+                    preloadAll: preloadAllAnimationData,
                     onlyFirstFrame: onlyFirstFrame)
             case .unknown:
                 image = Image(data: data, scale: scale)
             }
         #endif
-        
+
         return image
     }
+
+    @available(*, deprecated, message: "preloadAllGIFData is now preloadAllAnimationData")
```

Comment:
Since it is an internal API, we could just remove it instead of marking it as deprecated.